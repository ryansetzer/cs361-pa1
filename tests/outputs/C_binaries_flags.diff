--- outputs/C_binaries_flags.txt	2022-09-28 20:07:32.458378281 -0400
+++ expected/C_binaries_flags-4.txt	2022-09-28 17:44:42.825659551 -0400
@@ -5,14 +5,15 @@
 .hidden.txt
 empty.txt
 $ ./bin/ls -s data
-32 pwd.txt
-32 empty.txt
+9 pwd.txt
+0 empty.txt
 $ ./bin/ls -sa data
-24 .
-24 ..
-32 pwd.txt
-32 .hidden.txt
-32 empty.txt
+4096 .
+4096 ..
+9 pwd.txt
+5 .hidden.txt
+0 empty.txt
 $ ./bin/head -n 1 Makefile
+#
 $ quit
 
