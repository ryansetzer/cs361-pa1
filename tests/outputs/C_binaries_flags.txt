$ ./bin/ls -a data
.
..
pwd.txt
.hidden.txt
empty.txt
$ ./bin/ls -s data
32 pwd.txt
32 empty.txt
$ ./bin/ls -sa data
24 .
24 ..
32 pwd.txt
32 .hidden.txt
32 empty.txt
$ ./bin/head -n 1 Makefile
$ quit

