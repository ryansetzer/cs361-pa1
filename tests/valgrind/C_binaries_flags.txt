==59355== Memcheck, a memory error detector
==59355== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==59355== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==59355== Command: ../dukesh -b scripts/bins_flags.txt
==59355== 
==59355== 
==59355== HEAP SUMMARY:
==59355==     in use at exit: 0 bytes in 0 blocks
==59355==   total heap usage: 0 allocs, 0 frees, 0 bytes allocated
==59355== 
==59355== All heap blocks were freed -- no leaks are possible
==59355== 
==59355== For lists of detected and suppressed errors, rerun with: -s
==59355== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
