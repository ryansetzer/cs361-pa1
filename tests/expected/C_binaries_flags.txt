.
..
.hidden.txt
empty.txt
pwd.txt
0 empty.txt
9 pwd.txt
4096 .
4096 ..
5 .hidden.txt
0 empty.txt
9 pwd.txt
#
$ ./bin/ls -a data
$ ./bin/ls -s data
$ ./bin/ls -sa data
$ ./bin/head -n 1 Makefile
$ quit

